.drone.yml

```diff
@@ -76,9 +27,6 @@ steps:
     password:
       from_secret: docker_password
   when:
-    branch:
```

Comment:
Yeah, but now dependabot branches will crash here since docker hub doesn't allow a tag name to contain `/`

A quick and ugly way to fix this would be to use the commit hash as a docker tag instead of the branch name - but then we'd never be able to track those tags anymore, or remove them manually. The better way would be https://github.com/CRJI/EIC/issues/527 - so leave dependabot builds broken until that gets done